README.md

```diff
@@ -10,4 +10,13 @@ Giving out the bot's login secret is dangerous so you'll need to make your own b
 4. Run `npm start` to start testing
 5. If you want to play with gifs you need to get a [Giphy API key](https://developers.giphy.com) and add `GIPHY_KEY=yourgiphykey` to the .env file
 
-Follow [this guide](https://discordjs.guide/preparations/adding-your-bot-to-servers.html#bot-invite-links) to learn how to add a bot to your own server.
\ No newline at end of file
+Follow [this guide](https://discordjs.guide/preparations/adding-your-bot-to-servers.html#bot-invite-links) to learn how to add a bot to your own server.
+
+### Server configuration
+In order to use the bot properly in your own server there are 3 steps that you do:
+1. Have channels in your server named: `art,design,gameplay,music,level-design,production,writing,testland`. Include any or all of these channels, these are the channels people can build up a streak in. Currently this is hardcoded and you cannot have streak channels with other names. This is mandatory for a working bot.
```

Comment:
You don't have to include those channels specifically, they're the ones that come by default, see the db.js and how it initialises channels. Only announcements and testland are hardcoded.

Reply:
I added a comment on how to change the channels